src/components/User/User.tsx

```diff
@@ -1,36 +1,25 @@
 import React from 'react';
 
-import {UserAvatar} from '../UserAvatar';
-import type {UserAvatarSize} from '../UserAvatar';
-import type {QAProps} from '../types';
-import {block} from '../utils/cn';
+import {Avatar} from '../Avatar';
+import {UserWrapper} from '../UserWrapper';
 
-import './User.scss';
+import type {UserProps} from './types';
 
-const b = block('user');
+export const User = React.forwardRef<HTMLDivElement, UserProps>(
+    ({avatar, name, description, size, ariaAttributes, className, qa}, ref) => {
+        return (
+            <UserWrapper
+                avatar={avatar ? <Avatar {...avatar} size={size} /> : null}
+                name={name}
+                description={description}
+                size={size}
+                className={className}
+                ariaAttributes={ariaAttributes}
```

Comment:
Not sure if we need these props on such component

Reply:
It could be useful for using `User` with `Popup` or other components

Reply:
How?